documentation/source/install/install_usecases.rst

```diff
@@ -212,7 +216,7 @@ Run Jupyter Pod with OpenEBS Storage
 Use OpenEBS as persistent storage for the jupyter pod by selecting an OpenEBS storage class in the persistent volume claim. A sample jupyter pod yaml (with container attributes and pvc details) is available in the OpenEBS git repository (which was cloned in the previous steps).
 ::
    name@Master:~$ cat demo-jupyter-openebs.yaml
-
+   
```

Comment:
why is this extra line required?

Reply:
reformatted to remove lines. Since it is a snippet, i had 3 dots which warranted an extra line. Made it 2 dots and works without lines. Done!

Reply:
The setup link is moved to the prereq section. Overall flow any inputs?